src/tetris/brick/brickChances.ts

```diff
@@ -0,0 +1,83 @@
+
+export default class BrickChances {
+
+	//region public members
+	public get chances(): number[] {
```

Comment:
We could think about a better name. Otherwise it looks pretty stupid later on.

Reply:
Maybe probability?